Messaging/src/Messaging.sln

```diff
@@ -5,6 +5,20 @@ VisualStudioVersion = 16.0.29806.167
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RabbitMQWeb", "RabbitMQWeb\RabbitMQWeb.csproj", "{36602E04-9061-4828-B95F-C5D0472C768A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DependencyInjection", "Console\DependencyInjection\DependencyInjection.csproj", "{2FF4B674-B1D3-4D56-A759-6042CD12536C}"
```

Comment:
Are these all included in the repo? I don't see any other changes in this PR under `/Messaging/`

Reply:
So we did not add them to the solution for some reason - they were all there in the repo, just not in the solution.